src/app/components/user/cognito-user-view.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import {
+  func,
+} from 'prop-types';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import CheckBoxOutlineBlankOutlinedIcon from '@material-ui/icons/CheckBoxOutlineBlankOutlined';
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  ThemeProvider,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+
+import theme from '../../themes/theme';
+import {
+  user as userPropType,
+} from '../../shared/prop-types';
```

Comment:
A while ago I figured out how to avoid relative path imports, which helps when moving components around, as the relativity would have to change. I've made those changes to all the files I've found, and don't use relative path imports in new ones. Also, I tend to alphabetize the CHPL imports based on the "from" value, so this block would be:
```suggestion
import { user as userPropType } from 'shared/prop-types';
import { theme} from 'themes';
```